matrix-sdk/src/main/java/org/matrix/androidsdk/data/Room.java

```diff
@@ -183,6 +183,16 @@ public IMXStore getStore() {
         return mStore;
     }
 
+    /**
+     * Determine whether we should encrypt messages for invited users in this room.
+     *
+     * @return true if we should encrypt messages for invited users.
+     */
+    public boolean shouldEncryptForInvitedMembers() {
+        String historyVisibility = getLiveState().history_visibility;
```

Comment:
Sometimes we use `getState()` and sometimes we use `getLiveState()`. Actually `getLiveState()` is just an alias for `getState()`. We should consider keeping only one method for clarity.

Reply:
ok